src/rpcmisc.cpp

```diff
@@ -157,13 +157,14 @@ UniValue validateaddress(const UniValue& params, bool fHelp)
             "1. \"bitcoinaddress\"     (string, required) The bitcoin address to validate\n"
             "\nResult:\n"
             "{\n"
-            "  \"isvalid\" : true|false,         (boolean) If the address is valid or not. If not, this is the only property returned.\n"
```

Comment:
Please check spacing-alignment in the `bitcoin-cli help validateaddress`, not in the source code. These lines contain various numbers of escaped `"` characters...